Source/WebCore/html/HTMLPictureElement.cpp

```diff
@@ -75,5 +75,18 @@ bool HTMLPictureElement::isSystemPreviewImage()
 }
 #endif
 
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+bool HTMLPictureElement::isMultiRepresentationHEIC() const
+{
+    if (countChildNodes() != 2)
```

Comment:
why 2?

Reply:
`<source>` + `<img>`

Reply:
Surely there could be more than one `source` element? You can specify multiple `source` with different media queries. We should add a test for that case as well.

Reply:
Added a test case for multiple `source` with media queries.